pkg/tso/keyspace_group_manager_test.go

```diff
@@ -125,7 +125,7 @@ func (suite *keyspaceGroupManagerTestSuite) TestNewKeyspaceGroupManager() {
 // TestLoadKeyspaceGroupsAssignment tests the loading of the keyspace group assignment.
 func (suite *keyspaceGroupManagerTestSuite) TestLoadKeyspaceGroupsAssignment() {
 	re := suite.Require()
-	maxCountInUse := int(mcsutils.MaxKeyspaceGroupCountInUse)
+	maxCountInUse := int(mcsutils.MaxKeyspaceGroupCountInUse / 10)
```

Comment:
What's this change for?

Reply:
this is to fix a flaky test #6360 . I'll separate it.